proposals/description/tip_121c.ts

```diff
@@ -15,6 +15,16 @@ const DAO_TIMELOCK_LQTY_BALANCE = '1101298805118942906652299';
 const tip_121c: TemplatedProposalDescription = {
   title: 'TIP-121c: Final redemptions',
   commands: [
+    // 0. Unpause the DAI Holding PCV deposit
+    {
```

Comment:
Guardian will pause it before execution. Needs to be unpaused in this DAO vote so we can transfer funds from it to the psm